app/jobs/in_person/email_reminder_job.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
```

Comment:
starting a thread here arbitrarily for design comments @tomas-nava: 
1.  Can we shift which text we are linking in the bottom of the box outlining items to bring from "**Questions about what to bring**? Learn more" to "Questions? **Learn more about what to bring**." (indicating what text should be linked here with bold). Also [linking this design](https://www.figma.com/file/0ANJ9EFTj1nDpjep4oB1Kc/In-Person-Proofing-(Approved-%7C-Internal)?node-id=4343%3A11199&t=u9dEhPSOva5VhhBC-4) that accurately reflects what I mean. I also believe our link style uses an underline, if we could add that here.
2.  Are we able to add more space between the sentence "You can see any associate at the retail window to verify your identity" and "Your deadline to verify your identity in person is..." at the bottom of the email to more accurately reflect the design above as well?

Reply:
created [LG-8207](https://cm-jira.usa.gov/browse/LG-8207) to track this work